server/rooms.ts

```diff
@@ -19,63 +19,113 @@ const CRASH_REPORT_THROTTLE = 60 * 60 * 1000;
 
 const LAST_BATTLE_WRITE_THROTTLE = 10;
 
-/** @type {null} */
 const RETRY_AFTER_LOGIN = null;
 
-/** @type {typeof import('../lib/fs').FS} */
-const FS = require(/** @type {any} */('../.lib-dist/fs')).FS;
-const Roomlogs = require('./roomlogs');
+import {FS} from '../lib/fs';
+import {WriteStream} from '../lib/streams';
```

Comment:
Is this just being used as a type? I'd rather import it as a type if so.

```suggestion
type WriteStream = import('../lib/streams').WriteStream;
```

Reply:
Yes, its value is being used in `ladderIpLog`.